app/models/peep.rb

```diff
@@ -0,0 +1,15 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id,     Serial
+  property :text,  String
+
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [100/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)